ebos/eclproblem.hh

```diff
@@ -1999,7 +1999,21 @@ protected:
 
             // fluid-matrix interactions (saturation functions; relperm/capillary pressure)
             thermalLawManager_ = std::make_shared<EclThermalLawManager>();
-            thermalLawManager_->initParamsForElements(eclState, this->model().numGridDof());
+            std::function<std::vector<double>(const FieldPropsManager&, const std::string&, const unsigned int&)>
+            getDoubleOnLeaf = [this](const FieldPropsManager& fieldPropManager, const std::string& propString,
+                                  const unsigned int& numElems)
```

Comment:
Please fix indentation.

Reply:
It seems I forgot to correct indentation/delete-white spaces in the end. Thanks!